docs/requirements/product_backlog.md

```diff
@@ -0,0 +1,83 @@
+# Product Backlog
+
+## Epic *Core* (must have)
+
+> Description
+
+### Feature *REST API* (must have)
+
+> As a developer, I want to control the application via a REST API, in order to create a unified access for our application. 
+
+- Cost estimate: 20
+
+### Feature *CSAR Validation* (must have)
+> As a user, I want to get feedback whether my CSAR file is correct.
+
+
+#### Implementable Story *Syntax Validation* (must have)
+> As a user, I want to get feedback whether my CSAR file is correct in terms of syntax.
+
+- Cost estimate: 13
+
+#### Implementable Story *Semantic Validation* (could have)
+> As a user, I want to get feedback whether my CSAR file is correct in terms of semantics. 
```

Comment:
as we write below, we dont check the semantics but only report the error if exists.
semantic checks are quite difficult...